dags/tpu_observability/tpu_info_format_validation_dags.py

```diff
@@ -45,26 +40,23 @@ def get_tpu_info_from_pod(kubeconfig: str, pod_name: str) -> str:
   Returns:
     The standard output from the 'tpu-info' command.
   """
-  env = os.environ.copy()
-  env["KUBECONFIG"] = kubeconfig
+  with tempfile.NamedTemporaryFile() as temp_config_file:
+    env = os.environ.copy()
+    env["KUBECONFIG"] = temp_config_file.name
+
+    cmd = " && ".join([
+        jobset.Command.get_credentials_command(node_pool),
+        (f"kubectl exec {pod_name} -n default -- tpu-info"),
+    ])
+
+    result = subprocess.run_exec(
+        cmd,
+        env=env,
+        log_command=True,
+        log_output=True,
+    )
```

Comment:
```suggestion
    return subprocess.run_exec(cmd, env=env)
```
the default values of log_command and log_output are True already